openerp/addons/base/ir/ir_model.py

```diff
@@ -1049,10 +1049,21 @@ def _update(self,cr, uid, model, module, values, xml_id=False, store=True, noupd
                     },context=context)
         else:
             if mode=='init' or (mode=='update' and xml_id):
+                inherit_xml_ids = []
+                for table, field_name in model_obj._inherits.items():
+                    inherit_xml_id = '%s.%s' % (module, xml_id + '_' + table.replace('.', '_'))
```

Comment:
Standard code generates the xml id this way : https://github.com/OCA/OCB/pull/347/files#diff-678bd201dfe1a0c69add630cf73e76baR1070

I have to create exactly the same string, to detect if the value already exists and duplicated xml id creation.